index.js

```diff
@@ -1,12 +1,12 @@
-/* @flow */
+/* @flow strict */
 
 class CustomElementElement extends HTMLElement {
   constructor() {
     super()
   }
 
   connectedCallback() {
-    this.textContent = '<custom-element>'
```

Comment:
Why was this changed?

Reply:
Two reasons:

1. potentially confusing (there's no reason why this arbitrary string should look like a tag name
2. violates `github/unescaped-html-literal` from eslint-plugin-github